presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorMetadata.java

```diff
@@ -831,6 +832,66 @@ default ConnectorTableProperties getTableProperties(ConnectorSession session, Co
         return Optional.empty();
     }
 
+    /***
+     * Attempt to push down the aggregates into the table.
+     * <p>
+     * Connectors can indicate whether they don't support aggregate pushdown or that the action had no effect
+     * by returning {@link Optional#empty()}. Connectors should expect this method to be called multiple times
+     * during the optimization of a given query.
+     * <p>
+     * <b>Note</b>: it's critical for connectors to return Optional.empty() if calling this method has no effect for that
```

Comment:
Optional.empty() -> `{@code` or `{@link` it

Reply:
Done.